Editor/AGXUnityEditor/BrickUnity/BrickPrefabImporter.cs

```diff
@@ -71,15 +71,16 @@ public GameObject ImportFile(string filepath, string modelName)
       var b_simulation = new Brick.AGXBrick.BrickSimulation();
       b_simulation.AddComponent(b_component);
 
-      // TODO: if source filepath is within the Assets directory so should we set the rootpath to that.
       RootPath = "Assets";
       Name = b_component._ModelValuePath.Name.Str;
+      // Search subdirectories in Assets to see if the folder already exists somewhere, in another folder.
+      // Will change RootPath name if a folder is found.
+      FindExistingFolder();
```

Comment:
Minor detail: here there are mixed terminology; folder vs directory (see line below).

Reply:
Should be fixed now